src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {

```

Comment:
According to the checklist, class and interface names should not be verbs. The name 'AreaCalculator' suggests an action rather than an entity. Consider renaming the interface to something that represents an entity or ability, like 'AreaComputable' or 'Measurable', to better reflect its purpose.